.github/workflows/ssh.yml

```diff
@@ -0,0 +1,49 @@
+name: 'Manual Deploy on GPIO'
+
+on: [push]
+
+jobs:
+  GPIO:
+    name: SSH to GPIO
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: sshagent
+        uses: webfactory/ssh-agent@v0.5.2
+        with:
+          ssh-private-key: ${{ secrets.GPIO_SSHKEY }}
+
+      - name: 
+        run: |
+          branch="${GITHUB_REF#refs/heads/}"
+          echo "The branched that was pushed is $branch"
+          mkdir -p ~/.ssh
+          chmod 700 ~/.ssh
+          echo "github.planetfarms.io ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBDtrYtt44iGGRUDlL9aC3S/Xb40TuPx18NxngzjkSGrtq5tkPKQbOAZmEyVOONEFwHqIRQUgCbjQGwYtunbXaco=" > ~/.ssh/known_hosts
+          #ssh -o StrictHostKeyChecking=no root@github.planetfarms.io
+          ssh root@github.planetfarms.io <<'EOF'
+          cd /srv/planetfarms/
+          pwd
+          git branch
+          git checkout main
```

Comment:
Replace main with `$branch` from Line 21, but make sure to remove single quotes from `EOF` on Line 27 to enable bash substitution.